README.md

```diff
@@ -1,15 +1,25 @@
 
 # Etech_Recruitment_angular
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+## Getting Started
+
+To start project, clone this repo and run:
+
+`npm install`
+
+Then run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
 
 ## Code scaffolding
 
 Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|module`.
 
-## Build
+## Devlopment Build
+
+Run `ng build` or `ng build --dev` to build the project in development build, the api will be hit to development server. The build artifacts will be stored in the `dist/` directory.
+
+## Production Build
 
-Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `-prod` flag for a production build.
+Run `ng build --prod` to build the project in production build, the api will be hit to production/live server. The build artifacts will be stored in the `dist/` directory.
 
 ## Running unit tests
 
```

Comment:
also mention were the configuration files localted and what are the required configurations